src/urllib3/util/retry.py

```diff
@@ -191,8 +191,8 @@ class Retry:
     #: Default headers to be used for ``remove_headers_on_redirect``
     DEFAULT_REMOVE_HEADERS_ON_REDIRECT = frozenset(["Authorization"])
 
-    #: Maximum backoff time.
-    BACKOFF_MAX = 120
+    #: Default maximum backoff time.
+    DEFAULT_BACKOFF_MAX = 120
```

Comment:
This change makes sense, however I think we'd need a deprecation cycle in v1.26.x to remove `BACKOFF_MAX` safely in v2.0. Perhaps we can do that since it's likely not used often.

In 1.26.x I'm thinking we can go the route of other deprecated Retry classvars: https://github.com/urllib3/urllib3/pull/2000

Reply:
Ok, great!

To be sure I understand the process, I will open a PR to deprecate `BACKOFF_MAX`. The deprecation change will be included in `v1.26.x`.

Then this PR (including `DEFAULT_BACKOFF_MAX`) will be introduced in `2.0.0`.